package.json

```diff
@@ -55,11 +55,6 @@
       "docs/"
     ]
   },
-  "babel": {
```

Comment:
Looks like removing this for sure breaks `npm run dist` on minification. Adding it back in for me creates the error: 
`It looks like your Babel configuration specifies a module transformer. Please disable it.`

Can you run all things happily with this in JSON?